packages/core/src/config.ts

```diff
@@ -941,7 +941,7 @@ const composeDtsConfig = async (
 };
 
 const composeTargetConfig = (
-  target: RsbuildConfigOutputTarget = 'web',
+  target: RsbuildConfigOutputTarget = 'node',
 ): RsbuildConfig => {
```

Comment:
Could return in shape of `: { target: RsbuildConfigOutputTarget, rsbuildConfig: RsbuildConfig }`, so that we could use the unified resolved target. The duplicated fallback in line 717 can be removed then.

Reply:
great